### PR TITLE
Add persist-gatling-reports-to-mongodb container to the pod-kogito-gatling-benchmark.yaml

### PR DESCRIPTION
**Pull request by AlmogLevii**
### Description
Introduce a new initContsiner : `persist-gatling-reports-to-mongodb`
this new initContainer will run after the initContainer "upload-gatling-reports-to-nooba" 

# Note:
we need to add an entry to the secrets "obc-kogito-gatling-claim" with the key **MONGODB_CONNECTION_URI**
and the value something similar to `mongodb://username:password@10.88.0.1:27017`

### CODE REVIEW

File: test/open-shift/pod-kogito-gatling-benchmark.yaml [hunk 1]
```diff
@@ -34,3 +34,18 @@ spec:
       volumeMounts:
        - name: shared-file-storage
          mountPath: /src/kogito-gatling/target
+    - name: persist-gatling-reports-to-mongodb
+        image: quay.io/almoglevi0/persist-gatling-reports-to-mongodb
```
Comment (dmartinol) [inline test/open-shift/pod-kogito-gatling-benchmark.yaml:38]: I think you also need to describe how this image was generated (Dockerfile and related procedures)

Comment (AlmogLevii) [reply test/open-shift/pod-kogito-gatling-benchmark.yaml:38]: thanks, 
I described the build&run procedures in the separate (persist-gatling-reports-to-mongodb) project. 
let me know if more additions need to be done :)

Comment (dmartinol) [reply test/open-shift/pod-kogito-gatling-benchmark.yaml:38]: Do you mean it's in a separate git repository?

Comment (AlmogLevii) [reply test/open-shift/pod-kogito-gatling-benchmark.yaml:38]: yes [https://github.com/AlmogLevii/persist-gatling-reports-to-mongodb/blob/master/README.md](https://github.com/AlmogLevii/persist-gatling-reports-to-mongodb/blob/master/README.md)

Comment (dmartinol) [reply test/open-shift/pod-kogito-gatling-benchmark.yaml:38]: I see
Maybe, if we want to have an automatic generation of the testing system in the automation setup of the Middleware Performance team, we'd better integrate this repo inside the other one
Let's discuss this in the next meeting
Good job for now: can we have a demo tomorrow and compare the output with the data stored in Nooba to see they are consistent?

Comment (AlmogLevii) [reply test/open-shift/pod-kogito-gatling-benchmark.yaml:38]: ok, 
I still did not run the all containers sequence as we need if its what you mean (I can to give it a try tomorrow) 
but the project that I link before works well locally.

**Review by uteegozi — COMMENTED** [review]
you could change your base image inside your Dockerfile to the registry.access.redhat.com/ubi8/openjdk-11 since you are just running the app from there

**Comment by JudeNiroshan** [comment]
@uteegozi Thanks for this improvement. I have changed the source code. https://github.com/AlmogLevii/persist-gatling-reports-to-mongodb/commit/48e94e621a53c188cefa62ef215106e4953588af

@AlmogLevii please build the podman image locally and push it to quay.io with the same `latest` tag.

**Review by JudeNiroshan — APPROVED** [review]

**Pull request closed without merging**